optaweb-employee-rostering-backend/src/main/java/org/optaweb/employeerostering/domain/tenant/view/RosterConstraintConfigurationView.java

```diff
@@ -21,28 +21,72 @@
 import javax.validation.constraints.NotNull;
 
 import org.optaplanner.core.api.score.buildin.hardmediumsoftlong.HardMediumSoftLongScore;
+import org.optaplanner.persistence.jackson.api.score.buildin.hardmediumsoftlong.HardMediumSoftLongScoreJacksonDeserializer;
+import org.optaplanner.persistence.jackson.api.score.buildin.hardmediumsoftlong.HardMediumSoftLongScoreJacksonSerializer;
 import org.optaweb.employeerostering.domain.common.AbstractPersistable;
 
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
 public class RosterConstraintConfigurationView extends AbstractPersistable {
 
     @NotNull
     private DayOfWeek weekStartDay = DayOfWeek.MONDAY;
 
+    @JsonSerialize(using = HardMediumSoftLongScoreJacksonSerializer.class)
+    @JsonDeserialize(using = HardMediumSoftLongScoreJacksonDeserializer.class)
```

Comment:
Not sure about Jackson, but JSON-B has a type that reduces this boilerplate a lot. (`JsonB adapter`.)
Personally, I don't see a reason for dragging all five tons Jackson into this, when a small  JSON-B impl would suffice.

Reply:
This is purely for testing. For some reason RestAssured doesn't automatically pick up the OptaPlanner module (and registering it doesn't seem to work)

Reply:
If it's just for testing, that makes the boilerplate even worse. :-(

Reply:
Sadness: it was going to be fixed, but cannot since it will interfere with native tests: https://github.com/quarkusio/quarkus/pull/5430 . I'll see if moving to jsonb fixes it

Reply:
There a problem with JSON-B: It doesn't serialize map score values properly (it does serialize field score values properly).

Reply:
Maybe a custom adapter needs to be written? 
Anyway, since OptaPlanner persistence does have a JSON-B option, we should understand how to fix this.

Reply:
My guess is it doesn't register map adapters, so it uses the default adapter instead of using the toString adapter; creating test on OptaPlanner core

Reply:
I can confirm JSON-B doesn't work properly for maps in optaplanner-core (They get serialized as `{key: {}}` in tests)

Reply:
Moved the adapter registration to tests